app/routes/routes.py

```diff
@@ -0,0 +1,63 @@
+from flask import request, jsonify
```

Comment:
I like this pattern of common code for the routes in this one file while specific routes in this separate one. Very nice!